docs/cds/cone_search_query.rst

```diff
@@ -0,0 +1,206 @@
+.. _query_cone_search:
+
+Performing a CDS MOC query on a simple cone region
+--------------------------------------------------
+
+.. code:: ipython3
+
+    from astropy import coordinates
+    from astroquery.cds import cds
+
+
+We define a cone in astropy as the combination of a center position and
+a radius.
+
+.. code:: ipython3
+
+    center = coordinates.SkyCoord(10.8, 32.2, unit='deg')
+    radius = coordinates.Angle(1.5, unit='deg')
+
+The `CDS MOC Service`_ aims at returning all the data sets containing at
+least one observation located in the previously defined
+cone region.
+
+For that purpose, we will call the :meth:`~astroquery.cds.CdsClass.query_region` method of ``cds`` with
+the following parameters:
+
+.. code:: ipython3
+
+    cds.query_region(region_type=cds.RegionType.Cone,
+                     center=center,
+                     radius=radius)
+
+
+And we get an ID list of the data sets matching the query:
+
+.. parsed-literal::
+
+    ['CDS/B/assocdata/obscore',
+     'CDS/B/cb/lmxbdata',
+     'CDS/B/cfht/cfht',
+     'CDS/B/cfht/obscore',
+     'CDS/B/chandra/chandra',
+     'CDS/B/eso/eso_arc',
+     'CDS/B/gcvs/gcvs_cat',
+     'CDS/B/gcvs/nsv_cat',
+     'CDS/B/gemini/obscore',
+     ...]
+
+
+
+The ``region_type`` parameter is required. It informs the :meth:`~astroquery.cds.CdsClass.query_region`
+method about the type of region we are querying the `CDS MOC service`_ and
+thus constraints the following parameters of the method.
+
+You can find below the parameters associated to all the possible ``region_type`` parameter values :
+
+-  ``cds.RegionType.Cone`` :
+
+   -  center : `astropy.coordinates.SkyCoord <astropy.coordinates.SkyCoord>`
+        the center position of the cone
+   -  radius : `astropy.coordinates.Angle <astropy.coordinates.Angle>`
+        the radius of the cone
+
+-  ``cds.RegionType.Polygon`` :
+
+   -  vertices : [`astropy.coordinates.SkyCoord <astropy.coordinates.SkyCoord>`]
+        the positions forming the polygon
+
+-  ``cds.RegionType.Moc`` parameter value involves either :
+
+   -  filename : str
+        the local path to a fits file describing the MOC
+   -  url : str
+        an url to a fits file describing the MOC
+   -  moc : ``mocpy.MOC`` object
+        the region expressed directly as a MOC object
+
+An optional ``intersect`` parameter specifies the selection heuristic of
+the data sets. This parameter can take only three different values :
+
+- ``overlaps`` (default). The matching data sets are those overlapping the MOC region.
+- ``covers``. The matching data sets are those covering the MOC region.
+- ``enclosed``. The matching data sets are those enclosing the MOC region.
```

Comment:
Should it be `encloses` since the others are `covers` and `overlaps`?  For consistency.